src/core/utils/hooks/useOverflow.tsx

```diff
@@ -60,21 +69,26 @@ export const useOverflow = <T extends HTMLElement>(
     const availableWidth = containerRef.current.offsetWidth;
```

Comment:
Havent you ever noticed that offsetWidth is not giving you full width? I do remember ending up using `getoundingClientRect` and taking width from there

Reply:
Idk if I can to use it because `offsetWidht` and `scrollWidth` are rounded values so I can end up in a situation where client rect returns `300.5px` and `scrollWidth` is `301px` and then it would remove one item although it might be ok.